README.md

```diff
@@ -1,2 +1,227 @@
+
+Once your experiment is configured you are ready to run it using the ``qne experiment run`` command. Your experiment is parsed and sent to the NetSquid simulator. After some time your experiment run will be finished and a results directory will be generated in which all the results of your experiment are stored.
+
+
+## Prerequisites
+- A modern Linux or MacOS (10 or 11) 64-bit (x86_64) operating system. If you don’t have Linux or MacOS you could run it via virtualization, e.g. using VirtualBox. If you have Windows 10 or 11 you can also use the [Bash on Ubuntu](https://docs.microsoft.com/en-us/windows/wsl/) subsystem.
+- A [virtual environment](https://docs.python.org/3/library/venv.html) should be created and activated before creating an application.
+- Python version 3.7 or higher and pip version 19 or higher.
+- NetQASM makes use of SquidASM for which you need credentials in order to use it. These credentials can be obtained by registering on the forum of [NetSquid](https://forum.netsquid.org/).
+
+
+## Installation
+To install all the required packages, execute the following command:
+
+```
+pip install qne-adk
+```
+
+After installing the qne-cli, you can install SquidASM. Replace '{netsquid-user-name}' and '{netsquid-password}' with the credentials you registered on [NetSquid](https://forum.netsquid.org/):
+
+```
+pip install squidasm --extra-index-url=https://{netsquid-user-name}:{netsquid-password}@pypi.netsquid.org
+```
+
+Now everything should be setup and ready in order to create your own applications and experiments and run them on the simulator!
+
+
+## Commands
+The QNE-ADK uses various commands to create and run your applications and experiments. All of the commands are listed below:
+
+<!--- QNE APPLICATION CREATE --->
+<details closed>
+<summary><b>qne application create</b></summary>
+Create a new application in your current directory containing all the files that are needed to write your application. The application directory name will be based on the value given to <b>application</b>. Two child directories <b>src</b> and <b>config</b> will be created, along with the default files.
+<br></br>
+    
+```
+qne application create [OPTIONS] APPLICATION_NAME ROLES...
+
+Arguments:
+  APPLICATION_NAME  Name of the application  [required]
+  ROLES...          Names of the roles to be created  [required]
+
+Options:
+  --help  Show this message and exit.
+  
+Example:
+  qne application create application_name role_name1 role_name2
+```
+</details>
+
+
+<!--- QNE APPLICATION DELETE --->
+<details closed>
+<summary><b>qne application delete</b></summary>
+Used to delete an application. Will delete the entire application directory structure.
+<br></br>
+    
+```
+qne application delete [OPTIONS] APPLICATION_NAME
```

Comment:
APPLICATION_NAME is also optional. Why not copy the latest version from the current qne-adk?